src/components/Form.tsx

```diff
@@ -55,10 +56,15 @@ export const Form = ({
   const Buttons = (): JSX.Element => {
     function save(): void {
       if (selectedOption) {
-        console.log('saved'); // TODO Implement save callback
-        closeForm();
-        setIsSubButtonsVisible(false);
-        showAlert();
+        Create(selectedOption)
```

Comment:
can this be writtern as `async` `await` pattern for better readibility?

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/8/commits/cc17e6ff14033e2024085420713e9d47a63ac800, thank you